### PR TITLE
Component graphics v6 updates

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
Follow up to design issue https://github.com/patternfly/patternfly-org/issues/4249
and Closes https://github.com/patternfly/patternfly-org/issues/4249

### CODE REVIEW

File: packages/documentation-site/patternfly-docs/images/component-illustrations/accordian.png
Comment (evwilkin) [inline packages/documentation-site/patternfly-docs/images/component-illustrations/accordian.png:1]: Typo - this looks to be a duplicate of accordion.png except with a typo in the file name (accordian vs. accordion)

File: packages/documentation-site/patternfly-docs/images/component-illustrations/about-model.png
Comment (evwilkin) [inline packages/documentation-site/patternfly-docs/images/component-illustrations/about-model.png:1]: Duplicale of about-modal.png except with typo, this file can be deleted.

**Review by evwilkin — CHANGES_REQUESTED** [review]
These illustrations look great!  A few clarifications:
- `about-model.png` and `accordian.png` are duplicates with typos in the file names and can be deleted (the correctly-spelled versions do exist in this PR and are working correctly).
- `All components --splash --light.png` is the only net-new illustration but is not exported in the index file or used anywhere, is this supposed to be displayed somewhere?
- Prior illustrations were 500x500px, new illustrations are 1288 x 704px.  This aspect ratio works better to not make the cards/list rows unnecessarily tall, but the largest I am able to make the images display on the site is 639 x 319.5px (list view at 767px screen width).  The illustrations appear to be created at ~2x this resolution, confirming this is intentional for high dpi displays?  If not needed we can likely reduce file size with smaller images
Screenshot 2024-11-26 at 3 41 00 PM

**Comment by andrew-ronaldson** [comment]
The images were 4x scale so now they are back down to 2x. I removed the extra images that were misspelled and the ALL components image as it will be addressed in another issue.

**Comment by edonehoo** [comment]
image

Just noticed that it looks like some of the cards are full height, while some are more narrow. But as I adjust my window to be smaller, the full-height cards become narrow and the originally-narrow cards swap to full height 🤔

image

**Comment by evwilkin** [comment]
@edonehoo the cards are in each row should be the same height, and are set to match the height of the tallest card in that row - so in your second screenshot you'll see Application Launcher is extra tall to fit the "Demo" tag at the bottom so all other cards stretch so the height is universal across the row.  

As cards shift between rows, any rows without those extra tags don't need the extra space to accommodate them and so don't stretch vertically - so rows aren't a uniform height, but cards within a row should all match.  Hopefully that makes sense, but if it's still acting wonky or there's a different suggestion for how to handle that we can certainly revisit this logic 😄

**Comment by andrew-ronaldson** [comment]
I noticed a couple images that contained drop shadows were increase height on certain images. I don't think it will make a big difference but wanted them to be consistent

**Pull request merged**